emu_sv/utils.py

```diff
@@ -0,0 +1,10 @@
+import math
+import torch
+
+
+def _index_to_bitstring(vector: torch.Tensor, index: int) -> str:
```

Comment:
instead of passing a tensor here, pass `nqubits: int` directly: you are using the tensor just to get the shape, which is weird

Reply:
also, rename without the prefix underscore, since it is not private